test/base/MockFactory.scala

```diff
@@ -104,7 +104,7 @@ object MockFactory extends MockitoSugar {
   def buildMovementsMetricsMock: MovementsMetrics = {
     val movementsMetricsMock = mock[MovementsMetrics]
 
-    when(movementsMetricsMock.startTimer(any())).thenReturn(new Timer().time())
+    when(movementsMetricsMock.startTimer(anyString)).thenReturn(new Timer().time())
```

Comment:
You also can put type paramter to any `any[String]()`

Reply:
Just different approach